### DISCUSSION TITLE
New icons do not render properly in tmux

### DISCUSSION THREAD

**Discussion by AngelontheRoad** [ayamir/nvimdots#720] [Q&A]
I just merged #447 and updated all plugins. I tried to use the new nerd font with `Mono` and `Propo` and everything works perfectly without `tmux`. But inside `tmux` I found that icons were not rendered properly.

inside `tmux`:
![quicker_cf16e16a-d84d-40f0-a9a6-6af9aa1cebec.png](https://s2.loli.net/2023/05/07/omJXCP4kntrKSzx.png)

outside `tmux` is OK:
![quicker_58aeb3d9-cf82-4be2-a81f-4f79e6815336.png](https://s2.loli.net/2023/05/07/N4pc6OzRgGM2i8Q.png)

I found a [reddit post](https://www.reddit.com/r/tmux/comments/135ggtb/nerd_font_not_rendered_perfectly_in_tmux/) that has a similar problem, but no solution works for me.

By the way, I noticed that icons baseline is not a straight line (marked above).😥

Is there any help? Thanks

**Comment by charliie-dev** [comment]
r u using wezterm?

**Reply by AngelontheRoad** [reply]
I use alacritty

**Reply by charliie-dev** [reply]
three things u need to be careful with:
1. the `$TERM` set in your terminal and shell
2. the `$TERM` set for tmux
3. check the `$TERM` in nvim via `:!echo $TERM`

I would suggest set it all with `xterm-256color`.

https://github.com/CharlesChiuGit/dotfiles/blob/9f82f4b03684e54738571a4a6ae5e1dff92d3184/Tmux/.config/tmux/tmux.conf#L31-L35
https://github.com/CharlesChiuGit/dotfiles/blob/9f82f4b03684e54738571a4a6ae5e1dff92d3184/Zsh/.zsh/aliases.zsh#L18-L19
https://github.com/CharlesChiuGit/dotfiles/blob/9f82f4b03684e54738571a4a6ae5e1dff92d3184/Wezterm/.config/wezterm/wezterm.lua#L91-L93
https://github.com/CharlesChiuGit/dotfiles/blob/9f82f4b03684e54738571a4a6ae5e1dff92d3184/Zsh/.zshenv#L85-L95

I should probably set tmux's `$TERM` with `xterm-256color` lol, I'm testing sth in other day. Just letting u know u should be careful with that.

**Reply by AngelontheRoad** [reply]
My `tmux` term config is the same with yours, and I've tried to set `set-option -g default-terminal "screen-256color"` and nothing happened. I have worked with `tmux` properly until the merge, probably the render problem is caused by `tmux` or `neovim` themselves. Anyway, thanks for your reply :)

**Comment by ayamir** [comment]
Can confirm in `tmux` with `Propo` variant on `wezterm` with `xterm-256color` set.

**Comment by ayamir** [comment]
Also appears on other terminal multiplexer like `zellij`
